azure-iot-device/azure/iot/device/common/transport/pipeline_ops_base.py

```diff
@@ -16,8 +16,7 @@ class PipelineOperation(object):
     next stage.  If the operation gets to the end of the pipeline without being handled
     (completed), then it is treated as an error.
 
-    :ivar name: The name of the operation.  This, rather than the `isinstance` function, is
-      used by stages to decide if they want to handle on a particular operation object.
+    :ivar name: The name of the operation.  This is used primarily for logging
```

Comment:
>This is used primarily for logging [](start = 44, length = 34)

Why not just log with .__class__.__name__ then? Syntactical ease? I could understand that - it's pretty obscene to have to type that out every time we log (which is hopefully a lot)

Reply:
Syntactical ease, and it also gives us the option to be more explicit about names in the future.  I'm thinking about cases where we have 2 ops with the same name discriminated by namespace.  This allows us to add the namespace later if we want.

---
In reply to: [282122987](https://github.com/Azure/azure-iot-sdk-python-preview/pull/89#discussion_r282122987) [](ancestors = 282122987)